app/services/received_email_service.rb

```diff
@@ -139,8 +139,10 @@ def self.actor_from_email_and_group(email, group)
 
   def self.discussion_params(email)
     params = parse_route_params(email.route_path)
+    group = Group.find_by!(handle: (params['handle'] || email.route_path))
     {
-      group_id: Group.find_by!(handle: (params['handle'] || email.route_path)).id,
+      group_id: group.id,
+      private: !group.public_discussions_only?,
```

Comment:
Nice! Here you can use group.discussion_private_default

Reply:
ah.